cmd/cluster-agent-cloudfoundry/subcommands/run/command.go

```diff
@@ -3,179 +3,96 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build clusterchecks
-// +build clusterchecks
+//go:build !windows && clusterchecks
+// +build !windows,clusterchecks
```

Comment:
Was this automatically added?

Reply:
I noticed the build tags were inconsistent across all the DCA cloudfoundry files so I added these in some places where they were missing